courses/tests.py

```
@@ -1,3 +1,86 @@
+# Constant values to avoid repeated string literals
+COURSE_TITLE = "Advanced Python Course"
+COURSE_DESCRIPTION = "An in-depth Python course covering advanced topics."
+COURSE_LEVEL = "beginner"
+REQUIREMENT_TEXT = "Basic Python knowledge"
+OBJECTIVE_TEXT = "Master Django ORM"
+TEACHER_ROLE = "teacher"
+ENROLLMENT_STATUS = "pending"
+MODULE_TITLE = "Django Basics"
+
+class CourseModelTest(TestCase):
+    def setUp(self):
```

Ruff (0.8.2)

Missing return type annotation for public function `setUp`

Add return type annotation: `None`

(ANN201)